### PR TITLE
add faq about blocked extensions

### PR DESCRIPTION
**Pull request by thispaul**
add faq about blocked extensions to copilot docs

### CODE REVIEW

**Comment by ntrogh** [comment]
@isidorn can you also have a look at these changes?

File: docs/copilot/faq.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ Check each requirement if Copilot Chat doesn't work:
 - Make sure you have the latest version of both the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extensions.
 - Your GitHub account that is signed into VS Code must have an active Copilot subscription. Check your [Copilot subscription](https://github.com/settings/copilot).
 
+### Why is my Copilot Chat extension blocked?
+
+If you receive a message that a 3rd-party extension is blocked from using Copilot Chat, the extension was likely disabled due to a detected pattern of abuse coming from that specific extension. Please contact the developer of the extension (found on the VS Code Marketplace).
```
Comment (ntrogh) [inline docs/copilot/faq.md:92]:
```suggestion
If you receive a message that a third-party extension is blocked from using Copilot Chat, the extension was likely disabled due to a detected pattern of abuse coming from that specific extension. Contact the publisher of the extension when you encounter this issue. You can find the publisher information on the extension details page in the VS Code Marketplace.
```

**Review by ntrogh — CHANGES_REQUESTED** [review]
Slight rewording of the text.

**Comment by thispaul** [comment]
cc @bamurtaugh

**Comment by bamurtaugh** [comment]
Thanks @thispaul! LGTM, let's hear what @isidorn thinks too.

File: docs/copilot/faq.md [hunk 2]
```diff
@@ -87,6 +87,10 @@ Check each requirement if Copilot Chat doesn't work:
 - Make sure you have the latest version of both the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extensions.
 - Your GitHub account that is signed into VS Code must have an active Copilot subscription. Check your [Copilot subscription](https://github.com/settings/copilot).
 
+### Why is my Copilot Chat extension blocked?
+
+If you receive a message that a third-party extension is blocked from using Copilot Chat, the extension was likely disabled due to a detected pattern of abuse coming from that specific extension. Contact the publisher of the extension when you encounter this issue. You can find the publisher information on the extension details page in the VS Code Marketplace.
```
Comment (isidorn) [inline docs/copilot/faq.md:92]: Can we not say "third-party". We do not want it to sound like 1st parties we treat specially.

So I suggest to just say "extension".

Comment (isidorn) [inline docs/copilot/faq.md:92]: It is called "VS Marketplace", not "VS Code Marketplace"

**Review by isidorn — CHANGES_REQUESTED** [review]
Looks good overall. Request some minor changes and then we can merge-in.

File: docs/copilot/faq.md (hunk 2 above)
Comment (ntrogh) [inline docs/copilot/faq.md:92]:
```suggestion
If you receive a message that an extension is blocked from using Copilot Chat, the extension was likely disabled due to a detected pattern of abuse coming from that specific extension. Contact the publisher of the extension when you encounter this issue. You can find the publisher information on the extension details page in the Visual Studio Marketplace.
```
```suggestion
If you receive a message that a third-party extension is blocked from using Copilot Chat, the extension was likely disabled due to a detected pattern of abuse coming from that specific extension. Contact the publisher of the extension when you encounter this issue. You can find the publisher information on the extension details page in the VS Code Marketplace.
```

**Review by ntrogh — APPROVED** [review]

**Review by DonJayamanne — APPROVED** [review]

**Review by isidorn — APPROVED** [review]

**Pull request merged**